src/geometry/adjoints.jl

```diff
@@ -41,13 +41,22 @@ function ChainRulesCore.rrule(::Type{Interpolations.FilledExtrapolation}, itp, f
     return y, filledextra_const_pb
 end
 
-function ChainRulesCore.rrule(::Type{SVector{N, T}}, x...) where {N, T}
-    y = SVector{N,T}(x...)
-    function svector_const_pb(Δy)
+function ChainRulesCore.rrule(::Type{SArray{D, T, ND, L}}, x...) where {D, T, ND, L}
+    y = SArray{D, T, ND, L}(x...)
+    function sarray_pb(Δy)
         Δy = map(t->eltype(x...)(t...), Δy)
         return NoTangent(), Δy
     end
-    return y, svector_const_pb
+    return y, sarray_pb
+end
+
+function ChainRulesCore.rrule(p::Type{RotMatrix{N, T, L}}, x) where {N, T, L}
```

Comment:
Same here, don't declare types that you don't use.

```suggestion
function ChainRulesCore.rrule(p::Type{RotMatrix}, x)
```

Reply:
Same thing as I commented above.